config/manager/manager.yaml

```diff
@@ -47,6 +47,12 @@ spec:
       - effect: NoExecute
         key: node.alpha.kubernetes.io/unreachable
         operator: Exists
```

Comment:
should we remove those? seems those are alpha and per existing taints this need to be cleaned?

Reply:
I think we still need to keep this in order to make sure old k8s still works?

Reply:
the commit message say 'The old labels are removed.' ......
I agree it's reasonable to keep old things for backward compatible purpose

Reply:
Hmm. The taints were changed in 1.9 (https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.9.md#scheduling)

I think we safely could remove the old ones. What do you think @gyliu513 @jichenjc?

Reply:
it's removed in 1.9 so very unlikely someone will use it with cluster-api? I vote to remove since it's fairly old now..

Reply:
I agree, I see no reason for why we should support unsupported versions of k8s. :)

Reply:
+1 to remove as well